app/serializers/discussion_serializer.rb

```diff
@@ -38,6 +39,10 @@ def reader
     @reader ||= scope[:reader_cache].get_for(object) if scope[:reader_cache]
   end
 
+  def edited
+    object.versions.where(event: 'update').count > 1
```

Comment:
@gdpelican, having a bit of trouble with this, is it possible I need to implement something like the process outlined here: https://github.com/airblade/paper_trail#associations